component/org.wso2.carbon.identity.api.otp.service.smsOtp/pom.xml

```diff
@@ -0,0 +1,223 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>org.wso2.carbon.identity.api.otp.service</artifactId>
+        <groupId>org.wso2.carbon.identity.api</groupId>
+        <version>1.0-SNAPSHOT</version>
+        <relativePath>../../pom.xml</relativePath>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>org.wso2.carbon.identity.api.otp.service.smsotp</artifactId>
+    <packaging>war</packaging>
+    <name>WSO2 Identity Server - SMS-OTP Rest API</name>
+    <description>WSO2 Identity Server - SMS-OTP Rest API</description>
+
+    <dependencies>
```

Comment:
More readable if we maintain WSO2 dependencies and other separately.

Reply:
fixed in 69f620f